src/tutorials/0008-merkle-dags/05.md

```diff
@@ -0,0 +1,84 @@
+---
+    title: "Merkle DAGs: Verifiability"
+    type: "multiple-choice"
+---
+
+Because we use cryptographic-strength hash algorithms to create
+CIDs, they offer a high degree of *verifiability*: an individual
+who retrieves data using a content address can always compute the
+CID for themselves to ensure that they got what they were after.
+This offers both permanence (the data behind a content address
+will never change) and protection against malicious manipulation
+(bad actors can’t trick you into downloading the wrong file
+without you recognizing that it’s not the file you requested).
+
+In a Merkle DAG, the CID of each node depends, in turn, on the
+CIDs of each of its children. As such, the CID of a root node
+uniquely identifies not just that node, but the *entire* DAG of
+which it’s the root! As a result, we can extend all of the
```

Comment:
perhaps...?
```suggestion
which it is a root node! As a result, we can extend all of the
```

Reply:
I actually prefer this one as is  - contractions are less formal and we've already use node previously in the sentence so doesn't feel necessary to me.

Reply:
no objections!